Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityControllerMappingProfilesInspector.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using Microsoft.MixedReality.Toolkit.Core.Definitions.Devices;
+using Microsoft.MixedReality.Toolkit.Core.Services;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Core.Inspectors.Profiles
+{
+    [CustomEditor(typeof(MixedRealityControllerMappingProfiles))]
+    public class MixedRealityControllerMappingProfilesInspector : MixedRealityBaseConfigurationProfileInspector
```

Comment:
Still needs the ability to Add/Remove controller mapping profiles from the list

Reply:
I don't understand. We no longer have a list.

Reply:
Erm, it looks like a "list" and we do need one.
Almost like the Visualization screen, we should be able to add / remove mappings from the list to meet the requirements for projects, so they can only enable those controllers they want to support. (and easily add more)

Reply:
I got rid of the list as it was a lot harder to manage the it and prevent multiple from being registered, but I'll see what I can do.